neural_network/activation_functions/sigmoid_like.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+
+def sigmoid(vector: np.ndarray) -> np.ndarray:
+    """
+    The standard sigmoid function.
+    Args:
+        vector: (np.ndarray): The input array.
+    Returns:
+        np.ndarray: The result of the sigmoid activation applied to the input array.
+
+    >>> np.linalg.norm(np.array([0.5, 0.66666667, 0.83333333]) \
+        - sigmoid(vector=np.array([0, np.log(2), np.log(5)]))) < 10**(-5)
+    True
+    """
+    return 1 / (1 + np.exp(-1 * vector))
```

Comment:
```suggestion
    return 1 / (1 + np.exp(-vector))
```
Just slightly more concise